sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -309,6 +309,7 @@ functionCall
     | relevanceFunction                                             #relevanceFunctionCall
     | highlightFunction                                             #highlightFunctionCall
     | positionFunction                                              #positionFunctionCall
+    | GET_FORMAT LR_BRACKET functionArgs RR_BRACKET                 #getFormatFunctionCall
```

Comment:
Perhaps we should use something other than functionArgs since this is being interpreted as `QualifiedName`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/commit/bf2d9d2fd6a5a8176cd1df3371ec409676e8ccfb and https://github.com/Bit-Quill/opensearch-project-sql/commit/2dd0f84101be161ebecfe7daf900913b4b95d698